app/views/layouts/application.html.erb

```diff
@@ -9,11 +9,15 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Karla:ital,wght@0,200..800;1,200..800&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
 
   <body class="h-full bg-gray-8">
     <main class="relative flex flex-col w-full h-full mx-auto">
-      <% if !current_page?(new_session_path) %>
+      <% if !current_page?(new_session_path) && !current_page?(about_path) %>
```

Comment:
@LuigiR0jas  is there another way to make this simpler?

Reply:
I would suggest moving this into a helper